### PR TITLE
Avoid unnecessary calls to Yajl.dump() in out_loki.rb

### PR DESCRIPTION
**Pull request by nemobis**
Fixes issue #4061

### CODE REVIEW

**Comment by nemobis** [comment]
Just an example to show what line of the code I'm talking about, not tested. (It might also take a while before I can get the CLA signed by my employer.) Feel free to close.

**Comment by cyriltovena** [comment]
Looking good ! Let' see if you can get the CLA.

File: clients/cmd/fluentd/lib/fluent/plugin/out_loki.rb [hunk 1]
```diff
@@ -162,7 +162,10 @@ def write(chunk)
 
         res_summary = "#{res.code} #{res.message} #{res.body}"
         log.warn "failed to write post to #{@uri} (#{res_summary})"
-        log.debug Yajl.dump(body)
+        # Perform JSON conversion only if required by log level
+        if log.level <= 1
```
Comment (dannykopping) [inline clients/cmd/fluentd/lib/fluent/plugin/out_loki.rb:166]: It appears that fluentd offers constants for these log levels:
https://github.com/fluent/fluentd/blob/master/lib/fluent/log.rb#L37
Can you use that please?

Comment (nemobis) [reply clients/cmd/fluentd/lib/fluent/plugin/out_loki.rb]: Good point, it's much more readable this way. :)

**Review by dannykopping — APPROVED** [review]
Thanks @nemobis :+1:

**Comment by nemobis** [comment]
I hope to get the CLA signed next week (or when people are back from vacation).

**Comment by nemobis** [comment]
@dannykopping the CLA was signed on 2021-07-29, anything else left to do here? I see:

> You’re not authorized to merge this pull request.

**Pull request closed without merging**